src/load_inputs/load_resources_data.jl

```diff
@@ -26,16 +26,16 @@ end
 
 Internal function to get policy file information.
 
-# 
+# Returns
     policyfile_info (NamedTuple): A tuple containing policy file information.
 
 """
 function _get_policyfile_info()
     policyfile_info = (
-        esr     = (filename="Resource_energy_share_requirement.csv"),
-        cap_res = (filename="Resource_capacity_reserve_margin.csv"),
-        min_cap = (filename="Resource_minimum_capacity_requirement.csv"),
-        max_cap = (filename="Resource_maximum_capacity_requirement.csv"),
+        esr     = (filename="Resource_energy_share_requirement.csv", setup_param="EnergyShareRequirement"),
+        cap_res = (filename="Resource_capacity_reserve_margin.csv", setup_param="CapacityReserveMargin"),
+        min_cap = (filename="Resource_minimum_capacity_requirement.csv", setup_param="MinCapReq"),
+        max_cap = (filename="Resource_maximum_capacity_requirement.csv", setup_param="MaxCapReq"),
     )
     return policyfile_info
 end
```

Comment:
It makes me a bit nervous that policy files are being joined one-to-one with policy settings. While right now this is the case, it doesn't seem like a fundamental thing. What if some new policy requires two files?